### ISSUE TITLE
Support Django 3.2?

### ISSUE THREAD

**Issue by bluesurfer** [M1ha-Shvn/django-pg-bulk-update#72]
After upgrading to Django 3.2.3 I get the following error:

```
ImportError: cannot import name 'DefaultConnectionProxy' from 'django.db' 
```

**Comment by M1ha-Shvn** [comment]
Hi.
Sorry for long reaction, it looks like I've missed an email notification.
I'll have a look on this as soon as possible.

**Comment by M1ha-Shvn** [comment]
Released [version 3.4.3](https://github.com/M1ha-Shvn/django-pg-bulk-update/releases/tag/v3%2C4%2C3)
